code/residential-parking-permits/rpp.css

```diff
@@ -758,4 +758,64 @@ a.small-button {
 #kn-report-view_1369-1 div.highcharts-container svg g.highcharts-series-group g.highcharts-series rect:nth-child(10n+6){fill: #f15c80;}
 #kn-report-view_1369-1 div.highcharts-container svg g.highcharts-series-group g.highcharts-series rect:nth-child(10n+7){fill: #e4d354;}
 #kn-report-view_1369-1 div.highcharts-container svg g.highcharts-series-group g.highcharts-series rect:nth-child(10n+8){fill: #2b908f;}
-#kn-report-view_1369-1 div.highcharts-container svg g.highcharts-series-group g.highcharts-series rect:nth-child(10n+9){fill: #f45b5b;}
```

Comment:
This code didn't get deleted. Space was added after it, so it's rendering as delete in the diff. It's the same on the right side.

Reply:
looks like this extra line was added on line 211. I went ahead and removed it from the app css. Feel free to re-commit if you would like.

Reply:
@ChrispinP Thanks for fixing that in Knack production. The code in github now accurately reflects what we have in Knack.